frameworks/elastic/tests/test_tls.py

```diff
@@ -30,46 +33,69 @@ def service_account(configure_security):
 
 @pytest.fixture(scope='module')
 def elastic_service_tls(service_account):
-    sdk_install.install(
-        config.PACKAGE_NAME,
-        service_name=config.SERVICE_NAME,
-        expected_running_tasks=config.DEFAULT_TASK_COUNT,
-        additional_options={
-            "service": {
-                "service_account_secret": service_account["name"],
-                "service_account": service_account["secret"],
-                "security": {
-                    "transport_encryption": {
-                        "enabled": True
+    try:
+        sdk_install.install(
+            config.PACKAGE_NAME,
+            service_name=config.SERVICE_NAME,
+            expected_running_tasks=config.DEFAULT_TASK_COUNT,
+            additional_options={
+                "service": {
+                    "service_account_secret": service_account["name"],
+                    "service_account": service_account["secret"],
+                    "security": {
+                        "transport_encryption": {
+                            "enabled": True
+                        }
```

Comment:
The formatting of the dict looks off. (the `}` closing the `security` section.

Reply:
You sure? If you mean the above `}` it's closing `transport_encryption`.